examples/server/utils.hpp

```diff
@@ -642,9 +642,26 @@ static json oaicompat_completion_params_parse(
         throw std::runtime_error("Cannot use custom grammar constraints with tools.");
     }
 
+    /* Prefill assistant message support */
+    bool prefill_assistant_message = inputs.messages.size() > 0 && inputs.messages[inputs.messages.size()-1].role == "assistant";
+    common_chat_msg last_message;
+    if (prefill_assistant_message)
+    {
```

Comment:
```suggestion
    if (prefill_assistant_message) {
```

our code convention is `if () {` all in same line

Reply:
done